src/Icon/Icon.js

```diff
@@ -15,7 +15,9 @@ export const Icon = ({ glyph, size, clickHandler, className, ...props }) => {
         <span
             {...props}
             className={iconClasses}
-            onClick={clickHandler} />
+            onClick={clickHandler}
+            role='button'
```

Comment:
here (and elsewhere) should these just *be* buttons, rather than have role of button?

Reply:
Agreed.  I don't think we're setting a very good precedent by assigning `<span>` or `<div>` elements a role of `button`.  Because now we've have to wire up our own custom keyboard nav and potentially even focus management.  I know it makes this a more difficult (and time consuming) change.  But similar to parents, I think we need to set a good example.  😄